### PR TITLE
Deps/missing required deps

### PR DESCRIPTION
**Pull request by literat**
## Description

From this:

Screenshot 2024-09-06 at 17 52 59

To this:

Screenshot 2024-09-06 at 18 18 10

### CODE REVIEW

File: configs/stylelint-config-spirit/package.json [hunk 1]
```diff
@@ -30,5 +30,8 @@
     "stylelint": "16.7.0",
     "stylelint-order": "6.0.4",
     "stylelint-prettier": "5.0.0"
+  },
+  "devDependencies": {
+    "prettier": "^3.3.3"
```
Comment (literat) [inline configs/stylelint-config-spirit/package.json:35]: Dependency or Peer dependency? 🤔

**Comment by literat** [comment]
Screenshot 2024-09-06 at 21 16 43

A little bit better.

File: configs/stylelint-config-spirit/package.json (hunk 1 above)
Comment (literat) [reply configs/stylelint-config-spirit/package.json]: Dependency.

File: packages/analytics/package.json [hunk 2]
```diff
@@ -50,14 +50,17 @@
   },
   "devDependencies": {
     "@lmc-eu/eslint-config-jest": "3.0.2",
+    "@types/eslint": "^8",
     "@types/jest": "29.5.12",
     "@types/node": "20.14.9",
     "@typescript-eslint/eslint-plugin": "6.21.0",
     "@typescript-eslint/parser": "6.21.0",
+    "eslint": "8",
```
Comment (adamkudrna) [inline packages/analytics/package.json:58]: 🎉

File: packages/web-react/src/components/Item/demo/index.tsx [hunk 3]
```diff
@@ -1,3 +1,5 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
```
Comment (adamkudrna) [inline packages/web-react/src/components/Item/demo/index.tsx:2]: Will it work with the same syntax as on other lines?

```suggestion
// eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved
```

**Review by adamkudrna — APPROVED** [review]
Cool, thank you! ❤️

File: packages/web-react/src/components/Item/demo/index.tsx (hunk 3 above)
Comment (literat) [reply packages/web-react/src/components/Item/demo/index.tsx]: Fixed.

**Pull request merged**